.github/workflows/ci_cd.yml

```diff
@@ -159,6 +160,33 @@ jobs:
           # PRs and merges use STABLE_REVN
           # Workflow dispatch can use any revision number
 
+  test-container-info:
+    name: Get SHA of test container
+    runs-on: public-ubuntu-latest-8-cores
+    needs: [revn-variations]
+    strategy:
+      matrix:
+        test_container: ['${{ needs.revn-variations.outputs.test_container }}']
+    outputs:
+      sha: ${{ steps.get-sha.outputs.sha }}
+    steps:
+      - name: Login in Github Container registry
+        uses: docker/login-action@184bdaa0721073962dff0199f1fb9940f07167d1 # v3.5.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Pull, launch, and validate Mechanical service
+        env:
+          MECHANICAL_IMAGE: ${{ matrix.test_container }}
+        run: |
+          docker pull ${{ env.MECHANICAL_IMAGE }}
```

Comment:
There are ways to do this without pulling down the image (which takes time)

See  https://github.com/ansys/pyansys-geometry/blob/5bf5f74e5287882b9f0159084d85f58196846e51/.github/workflows/nightly_docker_test.yml#L59-L86